docs/README.md

```diff
@@ -133,6 +133,10 @@ Product Search:
     """
     hideUnavailableItems: Boolean = false
     """
+    If true, remove hidden facets from the result.
+    """
+    hidden: Boolean = false
```

Comment:
Could you choose a more specific name? Maybe `removeHiddenFacets` or something like that.

Reply:
okay, i will rename